app/src/main/res/layout/activity_main_player.xml

```diff
@@ -151,49 +151,66 @@
-                        android:scrollHorizontally="true"
-                        android:singleLine="true"
-                        android:textColor="@android:color/white"
-                        android:textSize="15sp"
-                        android:textStyle="bold"
+                        android:layout_marginTop="5dp"
+                        android:background="@drawable/ic_arrow_back_white_24dp"
+                        android:padding="10dp"
                         android:clickable="true"
                         android:focusable="true"
-                        tools:ignore="RtlHardcoded"
-                        tools:text="The Video Title LONG very LONG"/>
+                        android:scaleType="fitXY"
+                        android:tint="?attr/colorAccent"/>
 
-                    <TextView
-                        android:id="@+id/channelTextView"
-                        android:layout_width="match_parent"
+                    <LinearLayout
+                        android:id="@+id/metadataView"
+                        android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:ellipsize="marquee"
-                        android:fadingEdge="horizontal"
-                        android:marqueeRepeatLimit="marquee_forever"
-                        android:scrollHorizontally="true"
-                        android:singleLine="true"
-                        android:textColor="@android:color/white"
-                        android:textSize="12sp"
-                        android:clickable="true"
-                        android:focusable="true"
-                        tools:text="The Video Artist  LONG very LONG very Long"/>
+                        android:gravity="top"
+                        android:orientation="vertical"
+                        android:paddingLeft="8dp"
+                        android:paddingRight="8dp"
+                        tools:ignore="RtlHardcoded">
+
+                        <TextView
+                            android:id="@+id/titleTextView"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:ellipsize="marquee"
+                            android:fadingEdge="horizontal"
+                            android:marqueeRepeatLimit="marquee_forever"
+                            android:scrollHorizontally="true"
+                            android:singleLine="true"
+                            android:textColor="@android:color/white"
+                            android:textSize="15sp"
+                            android:textStyle="bold"
+                            android:clickable="true"
+                            android:focusable="true"
+                            tools:ignore="RtlHardcoded"
+                            tools:text="The Video Title LONG very LONG"/>
```

Comment:
What the hell is that? Is there any reason, why we put this here before? I guess, this is replaced by the correct title autmatically, but why do we use this as the default one 😂

Reply:
I think, that `tools:text` is only used by Android Studio to display a value in the layout preview. During runtime, the value will never be used.